user-manager/.github/workflows/gradle.yaml

```diff
@@ -0,0 +1,19 @@
+name: Java CI with Gradle
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: Build with Gradle
+        uses: gradle/gradle-build-action@bd5760595778326ba7f1441bcf7e88b49de61a25
+        with:
+          arguments: build
```

Comment:
you have to move your ci pipeline from user-manager folder to the root. Then it should work